CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ The changelog is available [on GitHub][2].
 * [#155](https://github.com/kowainik/relude/issues/155):
   Implement `Relude.Extra.Foldable` module.
 * Re-export `GHC.Float.atan2`.
+* Add `Monoid` and `Semigroup` instances for `Validation` type
```

Comment:
Thanks for updating CHANGELOG as well! Could you also write it in the format with the link to the corresponding issue number?

Reply:
Oops, sorry! I'll change this.